index.html

```diff
@@ -5422,7 +5640,7 @@ <h3 id="8_A_4">8.A.4 Example</h3>
                     <strong>Request</strong>
                 </p>
                 <pre>
-GET https://api.myparcel.nl/delivery_options?platform=myparcel&carrier=postnl&cc=be&carrier=1&number=49&postal_code=1000  HTTP/1.1
+GET https://api.myparcel.nl/delivery_options?platform=belgie&carrier=bpost&cc=be&number=49&postal_code=1000  HTTP/1.1

```

Comment:
waarom pas je dit aan? `platform=myparcel ` mag toch ook een voorbeeld hebben?

Reply:
Zie ook andere plekken. Ik vind het belangrijker dat nl voorbeelden heeft dan be.

Reply:
platform=myparcel heeft al een voorbeeld, die staat boven de platform=belgie